apps/frontend/src/app/3_organisms/StakeForm/hooks/useGetKickoffTs.ts

```diff
@@ -0,0 +1,22 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useGetProtocolContract } from '../../../../hooks/useGetContract';
+import { getRskChainId } from '../../../../utils/chain';
+
+export const useGetKickoffTs = () => {
+  const stakingContract = useGetProtocolContract('staking', getRskChainId());
+  const [kickoffTs, setKickoffTs] = useState(0);
+
+  const getKickoffTs = useCallback(async () => {
+    const getKickoffTs = await stakingContract?.kickoffTS();
```

Comment:
kickoff is a constant and same for everyone, we can cache it for a long time
```suggestion
    const result = await asyncCall('staking/kickoffTS', () => stakingContract?.kickoffTS(), { ttl: 1800_000 });
```